react/week3/todo-app/src/Components/AddTodo.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+
+
+function AddTodo({ addTodo }) {
+	const [value, setValue] = useState("");
+	const [deadline, setDeadline] = useState("");
+  const [date, setDate] = useState("");
```

Comment:
again  make sure to prettify the code,
move the code so all the states are in the same line